Content.Server/GameObjects/Components/Observer/GhostComponent.cs

```diff
@@ -139,6 +147,18 @@ private List<WarpPointComponent> FindWaypoints()
             return comp.EntityQuery<WarpPointComponent>().ToList();
         }
 
+        public void Examine(FormattedMessage message, bool inDetailsRange)
+        {
+            var mind = Owner.GetComponent<IActorComponent>().playerSession.ContentData()!.Mind;
+            var timeSinceDeath = _gameTimer.RealTime.Subtract(_timeOfDeath);
+            if(mind != null)
+            {
+                message.AddMarkup(Loc.GetString("Ghost of [color=white]{0}[/color].\nTime since death: [color=yellow]{1}[/color]",
```

Comment:
It probably doesn't need the "Ghost of name" part since it's already the title of the examine popup.
I like the time of death thing, but it can probably be formatted to just minutes since death.

Reply:
> It probably doesn't need the "Ghost of name" part since it's already the title of the examine popup

ha, dont know how i missed that.